tcms/kiwi_auth/views.py

```diff
@@ -30,12 +30,15 @@ class PasswordResetView(views.PasswordResetView):
 
 
 class Register(View):
-    """Register method of account"""
+    """
+    Register account view.
+    """
```

Comment:
I am wondering what warnings were you seeing wrt doc strings. As-is both of them are redundant b/c  they don't give any more information than the class/method name.

Reply:
Returned back before the change.

I haven't seen warnings. Just noticed that some of the class has comment on multiple lines, looks better,  because distinguish class comments from the method names.  Additionally before was a "register method" and now it's not a method anymore, but it's a class. It looks obvious, if you know what is doing, but it is not atm. We still have to provide docstring according to pylint so better have good one. Other alternative was to create a new commit.

I would even suggest this class to be renamed to: `UserRegistrationView` and form renamed too `UserRegistrationForm`, the same convention as Django form is called, i.e. both of them  having "Registration", "User" and what's about. 

Would also use "user" in the docstring, instead of "account" because it's mentioned "user" in this and other files.


What do you think?

Reply:
I think this is outside of scope for the moment.

Re pylint: We run pylint with `-d missing-docstring` so you are not supposed to be seeing any errors like you claim. If you are that means you are not running `make pylint` but another version, with other arguments.

Re naming: ATM we don't quite have a good naming convention for view classes. IMO adding "View" and "Form" at the end of the name when modules are also named in this way is redundant. The proper way to handle this is with a separate pylint plugin which will codify our naming policy. Something along these lines will happen for test classes around permissions. Can be extended in the future, but is also outside of scope for the current changes.